dev-haskell/uuid-types/uuid-types-1.0.3-r1.ebuild

```diff
@@ -18,10 +18,10 @@ KEYWORDS="~amd64 ~x86"
 IUSE=""
 
 RDEPEND=">=dev-haskell/binary-0.4:=[profile?] <dev-haskell/binary-0.9:=[profile?]
-	>=dev-haskell/random-1.0.1:=[profile?] <dev-haskell/random-1.3:=[profile?]
```

Comment:
When `RDEPEND` changes we usually have to bump ebuild's revision (from `1.0.3-r1` to `1.0.3-r2`) to force ebuild upgrade. Otherwise `portage` keeps to locally installed ebuild with it's upper bounds and produces very confusing errors on upgrade.

Reply:
Yes I'm aware of this and have been trying to do the rename as you've listed. If I've missed some it shouldn't be a problem at all to tweak it.